.devops/code-review-pipelines.yml

```diff
@@ -35,19 +35,20 @@ stages:
             displayName: 'newman installation'
 
           - script: |
-              newman run --ignore-redirects api-tests/checkout-tests/checkout-api-dev.tests.json --environment=api-tests/dev.envs.json --reporters cli,junit --reporter-junit-export Results/checkout-api-TEST.xml
+              newman run --ignore-redirects api-tests/checkout-tests/checkout-for-ecommerce-api.tests.json --environment=api-tests/dev.envs.json --reporters cli,junit --reporter-junit-export Results/checkout-api-TEST.xml
             displayName: 'Run API tests (DEV)'
             condition: and(succeeded(), eq('${{parameters.ENVIRONMENT}}', 'DEV'))
 
           - script: |
               newman run --ignore-redirects api-tests/checkout-tests/checkout-api-uat.tests.json --environment=api-tests/uat.envs.json --reporters cli,junit --reporter-junit-export Results/checkout-api-TEST.xml
+              newman run --ignore-redirects api-tests/checkout-tests/checkout-for-ecommerce-api.tests.json --environment=api-tests/uat.envs.json --reporters cli,junit --reporter-junit-export Results/checkout-api-TEST.xml
```

Comment:
Doesn't this ovewrite test reports for the PM API?

Reply:
Right, my bad.
Fixed with: https://github.com/pagopa/pagopa-checkout-tests/pull/49/commits/77f0646b99dfe97e0e202de478a1670f3d2532c6
See pipeline https://dev.azure.com/pagopaspa/pagoPA-projects/_build/results?buildId=87476&view=ms.vss-test-web.build-test-results-tab for execution results